portfolio/src/main/webapp/script.js

```diff
@@ -16,13 +16,15 @@
  * Adds a random greeting to the page.
  */
 function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+  const greetings = [
```

Comment:
Consider changing these variable names and comment above to facts since they are no longer greetings.

Reply:
Will Do Thanks for the catch